dev/index.d.ts

```diff
@@ -88,6 +88,11 @@ export declare function getDefaultWebpackConfig(options: DefaultWebpackOptions):
  */
 export declare function gulp_installAzureAccount(): Promise<void>;
 
+/**
+ * "Installs" a fake version of the resource groups extension before running tests. The extension isn't actually used for tests, but our extension would fail to activate without this
+ */
+export declare function gulp_installResourceGroups(): Promise<void>;
```

Comment:
nit: Aren't we referring to it as Azure Resources?

Reply:
For user-visible things, yeah I think we should call it "Azure Resources". For everything else, I think it's basically a free for all. There's too many things that will stay "azureresourcegroups" forever (extension id, github repo, etc.) so it's always going to be a bit confusing and not worth making consistent